src/libs/OptionsListUtils.js

```diff
@@ -775,13 +780,29 @@ function getCurrencyListForSections(currencyOptions, searchValue) {
  *
  * @param {Object} report
  * @param {Object} personalDetails
- * @returns {String}
+ * @returns {String[]}
```

Comment:
Ah good catch this was always returning an array. But the typical format we use for this is

```suggestion
 * @returns {Array<String>}
```

Reply:
Thanks @TomatoToaster :) I'm going to include it in the latest changes of today